packages/components/src/sandbox/index.js

```diff
@@ -142,6 +142,11 @@ class Sandbox extends Component {
 			body > div,
 			body > div > iframe {
 				width: 100%;
+			}
```

Comment:
Gloriously done. I haven't tested, but based on looking at the code this should work. 👍 👍